### DISCUSSION TITLE
Installation on A6X?

### DISCUSSION THREAD

**Discussion by dualice** [kudrykv/latex-yearly-planner#40] [General]
Hello,

How do you install these planners on a Supernote A6X?

Thank you,
Dualice.

**Comment by kudrykv** [comment]
Hello, @dualice!

You upload them to Document folder onto your device

**Reply by dualice** [reply]
Hello!!

Thank you! However, I tried that. Once I opened the document on my supernote, none of links inside the document.

**Reply by dualice** [reply]
AH.. I was trying to use the pen with the links. I have to use my finger! Thank you!
